azerothcore/azerothcore-wotlk
# Limiting password size

##### CHANGES PROPOSED:

-  Limit password size when changing username and when changing the password


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1545


##### TESTS PERFORMED:



##### HOW TO TEST THE CHANGES:


`account create testaccount 1234567890123456789`
It should return the error
`account create testaccount 123`
`account set password testaccount 1234567890123456789 1234567890123456789`
it should return the error again

Error that will be shown:
```
AC> account set password admin 1234567890123456789 1234567890123456789
AC>Your password can't be longer than 16 characters (client limit), password not changed!
```

##### KNOWN ISSUES AND TODO LIST:



##### Target branch(es):

Master
